pkg/api/v1/types.go

```diff
@@ -2110,6 +2110,8 @@ type PodSpec struct {
 	// If specified, the pod's scheduling constraints
 	// +optional
 	Affinity *Affinity `json:"affinity,omitempty" protobuf:"bytes,18,opt,name=affinity"`
+	// SchedulingMismatchedPredicateResults Summary of false podFitsOnNode predicates for investigating unscheduled pods.
```

Comment:
@smarterclayton , any thoughts ^ i can reimplement the struct if you still think its required.